packages/@aws-cdk/aws-certificatemanager/lib/certificate.ts

```diff
@@ -72,6 +72,13 @@ export interface CertificateProps {
    * @default CertificateValidation.fromEmail()
    */
   readonly validation?: CertificateValidation;
+
+  /**
+   * Enable or disable transparency logging for this certificate
+   *
+   * @default TransparencyLoggingPreference.ENABLED
+   */
+  readonly certificateTransparencyLoggingPreference?: TransparencyLoggingPreference;
```

Comment:
```suggestion
  readonly transparencyLoggingEnabled?: boolean;
```

Can we shorten the name to just `transparencyLoggingEnabled`? Also, since it is just enabled/disabled we can make it a `boolean`.

Reply:
Ok will do